htdocs/product/price.php

```diff
@@ -1439,7 +1440,12 @@ function on_change() {
 
     		// Il doit au moins y avoir la ligne de prix initial.
     		// On l'ajoute donc pour remettre a niveau (pb vieilles versions)
-    		$object->updatePrice($object->price, $object->price_base_type, $user, $newprice_min);
+    		//$object->updatePrice($object->price, $object->price_base_type, $user, $newprice_min);
+            if (! empty($conf->global->PRODUIT_MULTIPRICES)) {
+                $object->updatePrice($object->multiprices[1], $object->multiprices_base_type[1], $user, $object->multiprices_tva_tx[1], $object->multiprices_min[1], 1);
```

Comment:
@frederic34 
Why is the value 1 selected in the "multiprices" array?
`$object->multiprices[1]`

Reply:
the first price level

Reply:
@frederic34 
in an array the first is 0
$object->updatePrice($object->multiprices[0], $object->multiprices_base_type[0], ...

Reply:
![2018-02-05 - firefox developer edition](https://user-images.githubusercontent.com/3624836/35825541-7c35bc50-0ab6-11e8-8073-a9fb35ed974e.png)

Reply:
@frederic34 
arf ! sorry ! ;-)

Reply:
a page where prices are updated silently....